components/consumers/dependency-track/task.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 spec:
   description: Pushes findings to a Dependency-Track instance.
   params:
+    # Warning: at the time of writing this api-url is for the port 8081
     - name: consumer-dependency-track-api-url
       type: string
```

Comment:
Does it make sense to add the warning to the description? e.g. "Default port is 8081"

Reply:
not sure, this is only valid for local executions of DT, normally you'd not do that

Reply:
what does this mean? that the API URL is always concatenated with port 8081 if it doesn't have a port?